### PR TITLE
[stable/prometheus-operator] add relabelings/metricRelabelings to all exporter serviceMonitors

### PR DESCRIPTION
**Pull request by crandles**
Signed-off-by: Chris Randles <randles.chris@gmail.com>

#### What this PR does / why we need it:
Adds `metricRelabelings` and `relabelings` to all of the prometheus-operator exporter charts.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by vsliouniaev** [comment]
/lgtm
Thanks!

File: stable/prometheus-operator/templates/exporters/kube-api-server/servicemonitor.yaml [hunk 1]
```diff
@@ -14,6 +14,10 @@ spec:
     {{- end }}
     port: https
     scheme: https
+{{- if .Values.kubeApiServer.serviceMonitor.metricRelabelings }}
```
Comment (supremedacos) [inline stable/prometheus-operator/templates/exporters/kube-api-server/servicemonitor.yaml:17]: You created a metricRelabelings reference here but not in the `README.md`

File: stable/prometheus-operator/templates/exporters/kube-api-server/servicemonitor.yaml [hunk 2]
```diff
@@ -14,6 +14,10 @@ spec:
...
+    metricRelabelings:
+{{ toYaml .Values.kubeApiServer.serviceMonitor.metricRelabelings | indent 6 }}
+{{- end }}
 {{- if .Values.kubeApiServer.relabelings }}
```
Comment (supremedacos) [inline stable/prometheus-operator/templates/exporters/kube-api-server/servicemonitor.yaml:21]: In the `README.md` you reference this as `kubeApiServer.serviceMonitor.relabelings`

File: stable/prometheus-operator/templates/exporters/core-dns/servicemonitor.yaml [hunk 3]
```diff
@@ -21,4 +21,12 @@ spec:
     interval: {{ .Values.coreDns.serviceMonitor.interval }}
     {{- end }}
     bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+{{- if .Values.coreDns.serviceMonitor.metricRelabelings }}
```
Comment (supremedacos) [inline stable/prometheus-operator/templates/exporters/core-dns/servicemonitor.yaml:24]: You're missing references to these in the `README.md`

File: stable/prometheus-operator/templates/exporters/kube-api-server/servicemonitor.yaml [hunk 4]
```diff
@@ -14,6 +14,10 @@ spec:
     {{- end }}
     port: https
     scheme: https
+{{- if .Values.kubeApiServer.serviceMonitor.metricRelabelings }}
+    metricRelabelings:
+{{ toYaml .Values.kubeApiServer.serviceMonitor.metricRelabelings | indent 6 }}
```
Comment (supremedacos) [inline stable/prometheus-operator/templates/exporters/kube-api-server/servicemonitor.yaml:19]: For consistency should you have it `indent 4`?

File: stable/prometheus-operator/README.md [hunk 5]
```diff
@@ -356,15 +364,20 @@ The following tables list the configurable parameters of the prometheus-operator
...
 | `kubeStateMetrics.serviceMonitor.metricRelabelings` | Metric relablings for the `kube-state-metrics` ServiceMonitor | `[]` |
+| `kubeStateMetrics.serviceMonitor.relabelings` | The `relabel_configs` for scraping `kube-state-metrics`. | `` |
 | `kube-state-metrics.rbac.create` | Create RBAC components in kube-state-metrics. See `global.rbac.create` | `true` |
 | `kube-state-metrics.podSecurityPolicy.enabled` | Create pod security policy resource for kube-state-metrics. | `true` |
 | `nodeExporter.enabled` | Deploy the `prometheus-node-exporter` and scrape it | `true` |
 | `nodeExporter.jobLabel` | The name of the label on the target service to use as the job name in prometheus. See `prometheus-node-exporter.podLabels.jobLabel=node-exporter` default | `jobLabel` |
 | `nodeExporter.serviceMonitor.metricRelabelings` | Metric relablings for the `prometheus-node-exporter` ServiceMonitor | `[]` |
 | `nodeExporter.serviceMonitor.interval` | Scrape interval. If not set, the Prometheus default scrape interval is used | `nil` |
+| `nodeExporter.serviceMonitor.metricRelabelings` | The `metric_relabel_configs` for scraping the `prometheus-node-exporter`. | `` |
```
Comment (supremedacos) [inline stable/prometheus-operator/README.md:379]: This is already present 2 lines above

File: stable/prometheus-operator/templates/exporters/kube-api-server/servicemonitor.yaml (hunk 4 above)
Comment (crandles) [reply stable/prometheus-operator/templates/exporters/kube-api-server/servicemonitor.yaml:19]: 6 is right for this

**Comment by crandles** [comment]
Thanks, I'll fix the readme shortly

**Pull request merged**